README.md

```diff
@@ -79,6 +79,24 @@ az hanainstance monitor enable \
     --hana-db-password $HANA_DB_PW
 ```
 
+To restart a specific HANA instance:
+
+```
+az hanainstance restart --resource-group $RESOURCE_GROUP --instance-name $HANA_INSTANCE_NAME
```

Comment:
Open a separate PR to correct the fact that this wasn't already in the README

Reply:
Removed the restart part from README.